src/test/kotlin/org/jetbrains/research/transformations/util/PythonParserTest.kt

```diff
@@ -0,0 +1,13 @@
+package org.jetbrains.research.transformations.util
+
+import org.junit.jupiter.api.Test
+
+import org.junit.jupiter.api.Assertions.*
+
+internal class PythonParserTest {
+
+    @Test
+    fun ParserCheck() {
+        ParserSetup.checkSetup()
+    }
+}
```

Comment:
If it's a test, don't put it into `util` folder because this folder is used to contain some code, which is not a test code, but is used for tests like ParserSetup.
Also, what are you trying to test here? Usually, tests contain some assertions to check that everything works as expected. Just running ParserSetup is a little strange cause it only checks that no exceptions are raised. Honestly, I don't think we need tests for python parser setup because we'll see if it's not working when we try to run gumtree for python :)

Reply:
Thank you for the comment! the reason, that I created this file is that at the previous group meeting we discussed that this structure can take place here (as I understood then), but now I see, that without any "real" tests it is unnecessary indeed.  I've been running this test only to show, that no exceptions are raised. I will remove it in next commit.